### PR TITLE
Add k8s plugin

### PR DESCRIPTION
**Pull request by zc2638**
This is a more complete plugin that supports all k8s resources.

### CODE REVIEW

**Comment by zc2638** [comment]
Hi @d1wilko, 
CI seems to be cancelled, what should I do next?

File: plugins/k8s/content.yaml [hunk 1]
```diff
@@ -0,0 +1,99 @@
+title: Kubernetes
```
Comment (d1wilko) [inline plugins/k8s/content.yaml:1]: can we change this to k8s? - we already have a plugin titled Kubernetes and it is confusing when searching :(

**Review by d1wilko — CHANGES_REQUESTED** [review]
Looks good - just need one change :)

File: plugins/k8s/content.yaml (hunk 1 above)
Comment (zc2638) [reply plugins/k8s/content.yaml:1]: Thanks for your reply, but I have a confusion.
When I tested locally, I found that search has nothing to do with tags, only with title. 
So in order to make it better discovered, can I change it to `Kubernetes/k8s` or `Kubernetes Plugin`.

**Review by d1wilko — APPROVED** [review]
LGTM :) thanks

**Pull request merged**